core/storage/user/gae_models.py

```diff
@@ -55,25 +57,52 @@ class UserSettingsModel(base_models.BaseModel):
     # migration (to give role to all users) is run.
     role = ndb.StringProperty(
         required=True, indexed=True, default=feconf.ROLE_ID_EXPLORATION_EDITOR)
+    # When the user last agreed to the terms of the site. May be None.
+    last_agreed_to_terms = ndb.DateTimeProperty(default=None)
+    # When the user last logged in. This may be out-of-date by up to
+    # feconf.PROXIMAL_TIMEDELTA_SECS seconds.
+    last_logged_in = ndb.DateTimeProperty(default=None)
+    # Name of a user displayed in android UI. Unlike username, it can be
+    # edited and is unique only among the profiles of the account.
+    profile_name = ndb.StringProperty(default=None)
+    # User specified biography (to be shown on their profile page).
+    user_bio = ndb.TextProperty(indexed=False)
+    # User uploaded profile picture as a dataURI string. May be None.
+    profile_picture_data_url = ndb.TextProperty(default=None, indexed=False)
+    # Subject interests specified by the user.
+    subject_interests = ndb.StringProperty(repeated=True, indexed=True)
+    # When the user last edited an exploration.
+    # Exploration language preferences specified by the user.
+    # TODO(sll): Add another field for the language that the user wants the
```

Comment:
Btw, I think you can drop this TODO. We seem to have it below already and we just never removed the TODO.

Keep the second sentence though ("These language preferences ...") -- just append it to the sentence before this one. It adds useful context.

Reply:
Done.